action/rlp_tx_test.go

```diff
@@ -122,18 +116,18 @@ var (
 	rlpTests = []rlpTest{
 		{
 			"transfer",
-			"f86e8085e8d4a51000825208943141df3f2e4415533bb6d6be2a351b2db9ee84ef88016345785d8a0000808224c6a0204d25fc0d7d8b3fdf162c6ee820f888f5533b1c382d79d5cbc8ec1d9091a9a8a016f1a58d7e0d0fd24be800f64a2d6433c5fcb31e3fc7562b7fbe62bc382a95bb",
+			"",
```

Comment:
use `generateRLPTestRaw` func to generate raw txdata, instead of hardcoded string. will have a another PR to replace all hardcoded string in following test cases